src/react/components/pages/modelCompose/modelCompose.tsx

```diff
@@ -155,16 +155,16 @@ export default class ModelComposePage extends React.Component<IModelComposePageP
                 onColumnClick: this.handleColumnClick,
                 onRender: (model: IModel) => <span>{model.modelName}</span>,
             },
-            {
-                key: "column4",
-                name: strings.modelCompose.column.status.headerName,
-                fieldName: strings.modelCompose.column.status.fieldName,
-                minWidth: 50,
-                maxWidth: 100,
-                isResizable: true,
-                onColumnClick: this.handleColumnClick,
-                onRender: (model: IModel) => <span>{model.status}</span>
-            },
+            // {
+            //     key: "column4",
+            //     name: strings.modelCompose.column.status.headerName,
+            //     fieldName: strings.modelCompose.column.status.fieldName,
+            //     minWidth: 50,
+            //     maxWidth: 100,
+            //     isResizable: true,
+            //     onColumnClick: this.handleColumnClick,
+            //     onRender: (model: IModel) => <span>{model.status}</span>
+            // },
```

Comment:
remove

Reply:
The previous problem was to solve the problem that the generated URL contains multiple characters: / etc. now it is no longer needed. I will change it back

Reply:
I will